### ISSUE TITLE
im using it on bentobox 1blcok skyblock, doesnt seem to work, not sure..

### ISSUE THREAD

**Issue by tealtoxic** [AlessioGr/DisableVillagerBreeding#1]
hai! installed this plugin and placed the bypass permission on my shop for  my players to buy as access to breeding, but they still cant breed em villagers. once i removed the plugin, the baby villagers pops out like usual. 

even as admin, I too cant breed villager even with bypass. any idea?

**Comment by AlessioGr** [comment]
Hey, can you double-check you're using version 1.1 and that they have the permission `disablevillagerbreeding.bypass` (no typos)?

If it still doesn't work, I can imagine the server doesn't detect the player as a "breeder" and doesn't know who is responsible for the breeding. The plugin blocks all spontaneous breeding, no matter the permissions because it cannot check the permission if it doesn't know the breeder.

If that's the case, there's really no easy fix. I'd need to manually track the breeder via nbt tags/persistent data containers, which is gonna take a while to implement properly.

**Comment by tealtoxic** [comment]
yeah it is 1.1 and they have the bypass. i havent test in a non 1block skyblock environment so i can pinpoint either. but i will in the future. but im afraid i cant test it for you right now :) im just sharing what i experience. 

ill get back to you when i installed it again in another setup.
